forge-gui-mobile/src/forge/card/CardImageRenderer.java

```diff
@@ -146,7 +158,7 @@ public static void drawCardImage(Graphics g, CardView card, boolean altState, fl
         y += headerHeight;
 
         float artWidth = w - 2 * artInset;
-        float artHeight = artWidth / CardRenderer.CARD_ART_RATIO;
+        float artHeight = !showArtBox ? 0f : artWidth / CardRenderer.CARD_ART_RATIO;
```

Comment:
it will hide the art image if the selected border options in the settings is "Art", what happens on classic mode view and the deck editor views?